### PR TITLE
Update web to v0.7.1-alpha.0

### PR DESCRIPTION
**Pull request by mofojed**

### CODE REVIEW

File: web/client-ui/Dockerfile
Comment (mofojed) [inline web/client-ui/Dockerfile:5]:
```suggestion
ARG WEB_VERSION=0.7.1-alpha.0
```

**Review by devinrsmith — APPROVED** [review]
Approved - please hold off on merge until merge freeze is over.

**Comment by rcaudy** [comment]
Closing this, it has been incorporated into #1473

**Pull request closed without merging**
